recipes/r-easypar/meta.yaml

```
@@ -0,0 +1,109 @@
+{% set version = '1.0.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-easypar
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/caravagnalab/easypar/archive/refs/tags/v1.0.0.tar.gz
```

Use version definition

```suggestion
    - https://github.com/caravagnalab/easypar/archive/refs/tags/v{{ version }}.tar.gz
```